torch/_meta_registrations.py

```diff
@@ -1636,3 +1638,64 @@ def upsample_nearest2d_vec(input, output_size, scale_factors):
 import torch._refs
 import torch._refs.nn.functional
 import torch._refs.special
+
+
+def activate_meta():
+
+    activate_meta_table = {}
+
+    for type in ["meta", "post_autograd", "pre_autograd"]:
+        registry = global_decomposition_table[type]
+
+        for opo in registry:
+            if opo not in activate_meta_table:
+                activate_meta_table[opo] = registry[opo]
+
+    for op_overload, fn in activate_meta_table.items():
+        assert isinstance(op_overload, OpOverload)
+
+        op_overload.py_impl(torch._C.DispatchKey.Meta)(fn)
```

Comment:
@ezyang, 
I am wondering if we should make python dispatcher following the registration logic as c++ dispatcher. 

If yes, we will have the consistent behavior between python dispatcher and c++ dispatcher on Meta key.

Reply:
OK, will do in next PR.